app/Controllers/Http/StaticController.js

```diff
@@ -12,61 +12,19 @@ class StaticController {
   }) {
     return response.send({
       needToWaitToProceed: false,
-      isSpellcheckerPremiumFeature: false,
-      isSpellcheckerIncludedInCurrentPlan: true,
       isServiceProxyEnabled: true,
-      isServiceProxyIncludedInCurrentPlan: true,
-      isServiceProxyPremiumFeature: true,
-      isWorkspacePremiumFeature: false,
       isWorkspaceEnabled: true,
       isAnnouncementsEnabled: true,
       isSettingsWSEnabled: false,
       isServiceLimitEnabled: false,
       serviceLimitCount: 0,
-      isCommunityRecipesPremiumFeature: false,
-      isCommunityRecipesIncludedInCurrentPlan: true,
-      isCustomUrlIncludedInCurrentPlan: true,
       isMagicBarEnabled: true,
-      isTeamManagementIncludedInCurrentPlan: true,
       isTodosEnabled: true,
-      isTodosIncludedInCurrentPlan: true,
-      defaultTrialPlan: 'franz-pro-yearly',
       subscribeURL: 'https://ferdium.org',
-      planSelectionURL: 'https://ferdium.org',
       hasInlineCheckout: true,
       isPlanSelectionEnabled: false,
       isTrialStatusBarEnabled: false,
```

Comment:
Even this is not needed right ?? 

Above isPlanSelectionEnabled as well

Reply:
yes - will remove both as well